mks/model/stuf_02_04.py

```diff
@@ -238,7 +238,9 @@ def extract_verbintenis_data(persoon_tree: Tag):
 
         set_omschrijving_geslachtsaanduiding(result_verbintenis['persoon'])
 
-        set_reden_ontbinding_omschrijving_custom(result_verbintenis, verb.find('redenOntbinding').string)
+        einde_verbintenis_code = verb.find('redenOntbinding').string
+        if einde_verbintenis_code:
+            result_verbintenis['redenOntbindingOmschrijving'] = lookup_reden_ontbinding_partner.get(einde_verbintenis_code)
```

Comment:
Now you don't pass `None` anymore if there is no `einde_verbintenis_code`. Is that intended?

Reply:
when doing a .get(key, default_val) the default value defaults to `None`, so it is still there. the tests also cover this.

Reply:
I mean, if `eind_verbintenis_code` is None or empty string , `redenOntbindingOmschrijving` will not end up in the response.

```
tree.find('redenOntbinding').string = ''
        result = extract_data(tree)
        self.assertEqual('redenOntbindingOmschrijving' in result['verbintenis'], True) # <--- FAIL
```